web-app/src/components/tournament/details/TeamComponent.vue

```diff
@@ -6,43 +6,25 @@
     >
       <div
         v-if="inverted"
-        class="q-mx-md self-end text-right"
+        class="q-mr-xs text-right cut-name"
         :class="[textCenter ? 'self-center' : 'self-end']"
-        style="font-size: 15px;"
+        style="font-size: 15px; flex: 1"
       >
         {{ team.name }}
       </div>
 
-      <q-avatar
-        v-if="inverted"
-        class="self-end"
-        style="z-index: 10"
-        :size="iconSize + 'px'"
-      >
-        <!-- <img :src="team.members[0].avatarSrc" /> -->
-        <img src="https://cdn.quasar.dev/img/boy-avatar.png" />
-      </q-avatar>
-      <q-avatar
-        class="self-end"
-        :size="iconSize - 7 + 'px'"
-        :style="inverted ? 'margin-left: -8px' : 'margin-right: -8px'"
-      >
-        <!-- <img :src="team.members[1].avatarSrc" /> -->
-        <img src="https://cdn.quasar.dev/img/boy-avatar.png" />
-      </q-avatar>
-      <q-avatar
-        v-if="!inverted"
-        class="self-end"
-        style="z-index: 10"
-        :size="iconSize + 'px'"
-      >
-        <!-- <img :src="team.members[0].avatarSrc" /> -->
-        <img src="https://cdn.quasar.dev/img/boy-avatar.png" />
-      </q-avatar>
+      <multiple-users-avatar
+        :reverse="!inverted"
+        :maxAvatars="2"
+        :firstAvatarSize="iconSize"
+        :restAvatarsSize="iconSize - 6"
+        :users="team.members"
+        :center="textCenter"
+      />
 
       <div
         v-if="!inverted"
-        class="q-mx-md"
+        class="q-ml-xs cut-name"
         :class="[textCenter ? 'self-center' : 'self-end']"
         style="font-size: 15px;"
```

Comment:
same here. If using classes, use classes

Reply:
Did You mean to put all inline-styles to class? I used to use classes for few styles for one component or when this style is repeated in many places, but we can avoid all inline-styles